components/WithdrawLiquidity/WithdrawLiquidityController.tsx

```diff
@@ -156,10 +156,10 @@ const WithdrawLiquidity = () => {
         return
       }
 
-      if (provider) {
-        // Deposit poll contract address
-        let address = '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
+      // Deposit poll contract address
+      let address = process.env.DEPOSIT_POOL_ADDRESS || '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
```

Comment:
How does this work using || operator and &&?

Reply:
This will check if the process.env variable is found if it is undefined or null then any thing after || or && will be added to the address variable